### PR TITLE
change textareas to autoresize

### PR DESCRIPTION
**Pull request by bprize15**
resolves https://github.com/oncokb/oncokb-pipeline/issues/421

### CODE REVIEW

File: src/main/webapp/app/shared/firebase/input/RealtimeBasicInput.tsx [hunk 1]
```diff
@@ -139,6 +139,12 @@ const RealtimeBasicInput: React.FunctionComponent<IRealtimeBasicInput> = (props:
   const inputComponent = (
     <>
       <Input
+        innerRef={inputRef => {
+          if (inputRef && type === RealtimeInputType.TEXTAREA) {
+            inputRef.style.height = 'auto';
+            inputRef.style.height = `${inputRef.scrollHeight}px`;
```
Comment (zhx828) [inline src/main/webapp/app/shared/firebase/input/RealtimeBasicInput.tsx:145]: We overwrite the height twice?

Comment (bprize15) [reply src/main/webapp/app/shared/firebase/input/RealtimeBasicInput.tsx:145]: The first overwrite is for the case when the text box needs to shrink. When a row is deleted, the scroll height will not automatically change. That set to `auto` causes it to shrink to fit it's content.

There second one is for when the box grows. In that case the scroll height will automatically increase, and so I just need to set the height to the scroll height.

Comment (zhx828) [reply src/main/webapp/app/shared/firebase/input/RealtimeBasicInput.tsx:145]: Interesting, so basically after assigning auto, the inputref height is updated right away so you can get the height in the second assignment?

**Review by zhx828 — APPROVED** [review]
LGTM, the webdriver test is failing though.

**Review by calvinlu3 — APPROVED** [review]
Thanks for changes!

Just want to mention that in the `/diff` folder, there are two screenshots that I don't really see a diff.

**Pull request merged**